themes/mainroad/assets/css/style.css

```diff
@@ -1360,7 +1348,7 @@ summary:hover {
 	--color-code: #b12a48;
 	--color-code-background: #bdc7c7;
 	--color-code-notice: #e57;
-	--color-code-notice-background: #bdc7c7;
+	--color-code-notice-background: #fff;
```

Comment:
I assume this is the changed mentioned by:

> - Fixed the single-lined code block on notices to look unreadable in light mode.

Could you please point to an example of what this changes (preferrably, through a screenshot)?

Reply:
Before:
![image](https://github.com/neoforged/websites/assets/140878474/b788fd3a-679a-4e42-b226-770095846960)

After:
![image](https://github.com/neoforged/websites/assets/140878474/18af3fe8-cfe2-48b5-aa52-3bf697758adb)

Zoomed in, here, the code block may seem to be fine, but normally it's a little unreadable.

In all other places, the code blocks are unchanged.

Reply:
Perhaps reduce the brightness of the new background by a small bit? So it's not _pure_ white. Otherwise, sure.

Though, should we also change the background of all other code blocks, for consistency? 🤔